_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,129 @@
+When you use the `cockroach start-single-node` command to start a single-node cluster with Docker, additional features are available to help with testing and development.
+
+{{site.data.alerts.callout_danger}}
+Single-node clusters are not highly-available or fault-tolerant. They are not appropriate for production use.
+{{site.data.alerts.end}}
+
+- You can optionally set the following [Docker environment variables](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) to create a database and user automatically and to set a password for the user.
+  - `COCKROACH_DATABASE`
+  - `COCKROACH_USER`
+  - `COCKROACH_PASSWORD`
+
+  To prevent data loss, the environment variables are ignored if the `/cockroach/cockroach-data` directory within the container is not empty. To re-initialize the default database, user, and password from the environment variable values, delete the contents of `/cockroach/cockroach-data` within the running container.
+
+- You can optionally mount a directory of initialization scripts into the `docker-entrypoint-initdb.d` directory within the container. These scripts are run after CockroachDB starts and after the database and user (if specified as environment variables) have been created. The scripts run in the alphanumeric sort order imposed by your locale. The environment variables are ignored if the `/cockroach/cockroach-data` directory within the container is not empty.
+
+This section shows how to start a single-node cluster that uses these features.
+
+### Step 1. Create Docker volumes for the node
+
+Cockroach Labs recommends that you store cluster data in a Docker volume rather than in the storage layer of the running container. Otherwise, if a Docker container is inadvertently deleted, its data is inaccessible.
+
+To create the [Docker volume](https://docs.docker.com/storage/volumes/) where the cluster will store its data:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+docker volume create roach-single
+~~~
+
+### Step 2. Start the cluster
```

Comment:
The formatting of the substeps is broken.

<img width="1088" alt="Screen Shot 2022-09-29 at 2 43 05 PM" src="https://user-images.githubusercontent.com/20482960/193147526-ad0f4130-95a1-4cd8-8063-d296d2c43218.png">

Reply:
Re-implemented the sub-bullets as HTML and captured the sub-note in a Liquid variable.